libs/datamodel/core/tests/capabilities/mod.rs

```diff
@@ -55,8 +55,9 @@ fn multiple_indexes_with_the_same_name_only_supported_if_all_specified_providers
     // Only MySQL supports multiple indexes with the same name.
```

Comment:
comment can now also be updated :D

Reply:
Just following the tradition of making comments not useful here... We've done that for centuries!